client/elements/sc-page-static.html

```diff
@@ -109,13 +127,39 @@
     </style>
 
     <app-toolbar>
-      <div class="title" main-title><h1 class="title-text">SuttaCentral</h1></div>
+      <div class="title" main-title>
+        <h1 class="title-text">SuttaCentral</h1>
+      </div>
     </app-toolbar>
     <app-toolbar id="lasttoolbar">
       <div class="title" main-title>
         <p class="subtitle">Early Buddhist texts, translations, and parallels</p>
       </div>
     </app-toolbar>
+    <app-toolbar id="nav_toolbar" bottom-item>
+      <paper-tabs id="nav_tabs" selected="0" scrollable hide-scroll-buttons>
+        <paper-tab class="nav-link" data-name="HOME">
+          <div class="link-text">HOME</div>
+          <a href="/home" class="hidden-link"></a>
+        </paper-tab>
+        <paper-tab class="nav-link" data-name="INTRODUCTION">
+          <div class="link-text">INTRODUCTION</div>
+          <a href="/introduction" class="hidden-link"></a>
+        </paper-tab>
+        <paper-tab class="nav-link" data-name="DONATIONS">
+          <div class="link-text">DONATIONS</div>
+          <a href="/donations" class="hidden-link"></a>
+        </paper-tab>
+        <paper-tab class="nav-link" data-name="OFFLINE">
+          <div class="link-text">USE OFFLINE</div>
+          <a href="/offline" class="hidden-link"></a>
```

Comment:
Why did you use this `hidden-link` class? The documentation has examples for links in this element. Did the following not work?

```
<paper-tabs selected="0">
  <paper-tab link>
    <a href="#item1" tabindex="-1">ITEM ONE</a>
  </paper-tab>
  <paper-tab link>
    <a href="#item2" tabindex="-1">ITEM TWO</a>
  </paper-tab>
  <paper-tab link>
    <a href="#item3" tabindex="-1">ITEM THREE</a>
  </paper-tab>
</paper-tabs>
```